tests/phpunit/integration/tests/Story_Revisions.php

```diff
@@ -102,4 +104,63 @@ public function data_test_revisions_to_keep(): array {
 			],
 		];
 	}
+
+	public function test_filter_revision_fields_not_an_array(): void {
+		$this->assertSame( 'foo', $this->instance->filter_revision_fields( 'foo', [] ) );
+	}
+
+	public function test_filter_revision_fields_wrong_post_type(): void {
+		$fields = [
+			'post_title' => 'Post title',
+		];
+
+		$this->assertSame(
+			$fields,
+			$this->instance->filter_revision_fields(
+				$fields,
+				[
+					'post_type'   => 'post',
+					'post_parent' => 0,
+				]
+			)
+		);
+	}
+
+	public function test_filter_revision_fields_story_post_type(): void {
+		$fields = [
+			'post_title' => 'Post title',
+		];
+
+		$actual = $this->instance->filter_revision_fields(
+			$fields,
+			[
+				'post_type'   => Story_Post_Type::POST_TYPE_SLUG,
+				'post_parent' => 0,
+			]
+		);
+
+		$this->assertArrayHasKey( 'post_content_filtered', $actual );
+	}
+
+	public function test_filter_revision_fields_story_post_type_revision(): void {
```

Comment:
Add cover notiations.

Reply:
Good call! Just noticed that the `coversDefaultClass` annotation was wrong here.